react/src/lib/components/DeckGLMap/layers/terrain/map3DLayer.ts

```diff
@@ -326,9 +358,19 @@ export default class Map3DLayer extends CompositeLayer<
 > {
     initializeState(): void {
         // Load mesh and texture and store in state.
+        const isBounds = typeof this.props.bounds !== "undefined";
+        if (isBounds) {
```

Comment:
Should also warn of deprecation if `rotDeg` or `rotPoint` is defined outside the `frame` prop.